ui/src/app/components/runtime-configuration-panel.enzyme.spec.tsx

```diff
@@ -398,106 +330,20 @@ describe(RuntimeConfigurationPanel.name, () => {
-
-      const wrapper = await component();
-
-      expect(getMainCpu(wrapper)).toEqual(findMachineByName(machineType).cpu);
-      expect(getMainRam(wrapper)).toEqual(
-        findMachineByName(machineType).memory
-      );
-      expect(getDetachableDiskSize(wrapper)).toEqual(persistentDisk.size);
+      // Moved to RTL
     }
   );
 
   it(
     'should set runtime preset values in customize panel instead of getRuntime values ' +
       'if configurationType is HailGenomicsAnalysis',
     async () => {
-      const customMasterMachineType = 'n1-standard-16';
-      const customMasterDiskSize = 999;
-      const customWorkerDiskSize = 444;
-      const customNumberOfWorkers = 5;
-      setCurrentRuntime({
-        ...runtimeApiStub.runtime,
-        status: RuntimeStatus.DELETED,
-        configurationType: RuntimeConfigurationType.HAIL_GENOMIC_ANALYSIS,
-        gceConfig: null,
-        gceWithPdConfig: null,
-        dataprocConfig: {
-          ...defaultDataprocConfig(),
-          masterMachineType: customMasterMachineType,
-          masterDiskSize: customMasterDiskSize,
-          workerDiskSize: customWorkerDiskSize,
-          numberOfWorkers: customNumberOfWorkers,
-        },
-      });
-
-      // show that the preset values do not match the existing runtime
-
-      const {
-        masterMachineType,
-        masterDiskSize,
-        workerDiskSize,
-        numberOfWorkers,
-      } = runtimePresets.hailAnalysis.runtimeTemplate.dataprocConfig;
-
-      expect(customMasterMachineType).not.toEqual(masterMachineType);
-      expect(customMasterDiskSize).not.toEqual(masterDiskSize);
-      expect(customWorkerDiskSize).not.toEqual(workerDiskSize);
-      expect(customNumberOfWorkers).not.toEqual(numberOfWorkers);
-
-      const wrapper = await component();
-      await mustClickButton(wrapper, 'Customize');
-
-      expect(getMainCpu(wrapper)).toEqual(
-        findMachineByName(masterMachineType).cpu
-      );
-      expect(getMainRam(wrapper)).toEqual(
-        findMachineByName(masterMachineType).memory
-      );
-      expect(getMainDiskSize(wrapper)).toEqual(masterDiskSize);
-      expect(getWorkerDiskSize(wrapper)).toEqual(workerDiskSize);
-      expect(getNumWorkers(wrapper)).toEqual(numberOfWorkers);
+      // Moved to rtl
     }
   );
 
   it('should reattach to an existing disk by default, for deleted VMs', async () => {
-    const disk = existingDisk();
-    setCurrentDisk(disk);
-    setCurrentRuntime({
-      ...runtimeApiStub.runtime,
-      status: RuntimeStatus.DELETED,
-      configurationType: RuntimeConfigurationType.USER_OVERRIDE,
-      gceConfig: {
-        ...defaultGceConfig(),
-        machineType: 'n1-standard-16',
-      },
-      dataprocConfig: null,
-    });
-
-    const wrapper = await component();
-    expect(getDetachableDiskSize(wrapper)).toEqual(disk.size);
+    // Moved to RTL
   });
 
   it('should allow configuration via dataproc preset from modified form', async () => {
```

Comment:
mark this and the following 2 tests as converted

Reply:
Done